### PR TITLE
Add BCN hackathon page

### PR DESCRIPTION
**Pull request by christopher-hakkaart**

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

File: sites/main-site/src/content/events/2024/hackathon-barcelona.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: Hackathon - May 2024 (Barcelona)
+subtitle: An in-person hackathon held in Barcelona
+type: hackathon
+announcement:
+    start: 2024-10-28T09:00:00+02:00
```
Comment (mashehu) [inline sites/main-site/src/content/events/2024/hackathon-barcelona.mdx:6]: are you sure you don't want to announce it earlier?

File: sites/main-site/src/content/events/2024/hackathon-barcelona.mdx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+# Socials activities
+
+During the hackathon, we will have light-hearted fun and games!
+Special prizes are up for grabs for the winners!
+
+_More details will be available soon._
+
+# Venue: World Trade Center Barcelona
+
+The hackathon will be taking place in at the [World Trade Center Barcelona](https://maps.app.goo.gl/Fyezb8CiS11wEGAU6).
```
Comment (mashehu) [inline sites/main-site/src/content/events/2024/hackathon-barcelona.mdx:50]:
```suggestion
The hackathon will be taking place at the [World Trade Center Barcelona](https://maps.app.goo.gl/Fyezb8CiS11wEGAU6).
```

**Review by mashehu — APPROVED** [review]
Should we add a warning that this is not a training event and prior knowledge is required, or will there actually be again a training in parallel?

File: sites/main-site/src/content/events/2024/hackathon-barcelona.mdx (hunk 1 above)
Comment (christopher-hakkaart) [reply sites/main-site/src/content/events/2024/hackathon-barcelona.mdx]: Noted and fixed - good shout!

**Review by maxulysse — APPROVED** [review]

**Pull request merged**